news/backends/sfdc.py

```diff
@@ -94,6 +94,10 @@ def to_vendor(data):
         # True for every contact moving through basket
         'Subscriber__c': True,
     }
+    if 'email' in data:
+        for domain in settings.TESTING_EMAIL_DOMAINS:
+            if data['email'].endswith(u'@{}'.format(domain)):
+                contact['UAT_Test_Data__c'] = True
```

Comment:
Yes. It's a field they added and is false by default. We could set it that way here, but I think that could potentially overwrite the value for accounts they're testing for which they manually checked the box.